integration_tests/mockApis/visitScheduler.ts

```diff
@@ -166,20 +166,25 @@ export default {
   stubVisit: (reference: string): SuperAgentRequest => {
     const result: Visit = {
       applicationReference: 'aaa-bbb-ccc',
-      reference: 'ab-cd-ef-gh',
+      reference: 'gh-ef-cd-ab',
```

Comment:
This works fine, but it would be more flexible if `stubVisit` didn't have hard-coded data. and instead took a visit param (e.g. `stubVisit: (visit: Visit): SuperAgentRequest => ...`) and then returned it (`... jsonBody: visit,`)

In the test script you'd then do something like:
```
const visit = TestData.visit({ // .. any options .. //})
cy.task('stubVisit', visit)
```

More re-usable then; e.g. when moving on to cancellation journey; could stub that with:
```
const cancelledVisit = TestData.visit({ visitStatus: 'CANCELLED' })
cy.task('stubVisit', cancelledVisit)
```